contracts/Vault.vy

```diff
@@ -204,6 +378,16 @@ def setWithdrawalQueue(_queue: address[MAXIMUM_STRATEGIES]):
 
 @internal
 def _transfer(_from: address, _to: address, _value: uint256):
+    """
+    @notice
+        Transfers `_value` shares from `_from` to `_to`. This call will fail
+        if the _to address is this contract's address, or if `_to` is 0x0.
+    @param _from The address shares are being transfered from.
+    @param _to 
+        The address shares are being transfered to. Must not be this contract's
+        address, must not be 0x0.
+    @param _value The quantity of shares to transfer.
+    """
```

Comment:
Not sure if `internal` functions should have NatSpec, does not appear in output at least.

Reply:
They don't appear, I was going off the NatSpec docs, "The Solidity compiler only interprets tags if they are external or public. You are welcome to use similar comments for your internal and private functions, but those will not be parsed.", and just putting them in for consistency / for someone reading the code.

I can leave as-is, change to normal comments, or remove entirely, let me know your preference.

Reply:
Updated, per discussing with @fubuloubu in Telegram I've adjusted the internal functions to use simplified comments instead of NatSpec.